docs/custom-prompts.md

```diff
@@ -4,14 +4,17 @@ With Enquirer 2.0, custom prompts are easier than ever to create and use.
 
 **How do I create a custom prompt?**
 
-Custom prompts are created by extending Enquirer's `Prompt` class, or one of the built-in [prompts](#-prompts) or low-level [types](#-types).
+Custom prompts are created by extending either:
+- Enquirer's `Prompt` class
+- one of the built-in [prompts](#-prompts), or 
+- low-level [types](#-types).
 
-<!-- Example: HaiKarate Custom Prompt -->
+<!-- Example: MyCustomPrompt Custom Prompt -->
```

Comment:
This is a joke about [Hai Karate](https://en.wikipedia.org/wiki/Hai_Karate) cologne. Please revert these changes, but I like the other changes.

Reply:
I'm sorry. I don't know about that. 'MyCustomPrompt' was generic.
Will revert back.